examples/import_export/import-ecdh-key.js

```diff
@@ -18,46 +18,74 @@ export default async function () {
       208, 131, 136, 154, 125, 18, 20, 202, 231, 168, 184, 127, 53, 186, 6, 136,
       114, 101, 127, 109, 179, 44, 96, 108, 193, 126, 217, 131, 163, 131, 135,
     ])
-  );
+  ).then(
+    (aliceKeyPair) => {
+      importKeys(
+        new Uint8Array([
+          4, 218, 134, 37, 137, 90, 68, 101, 112, 234, 68, 87, 110, 182, 85,
+          178, 161, 106, 223, 50, 150, 9, 155, 68, 191, 51, 138, 185, 186, 226,
+          211, 25, 203, 96, 193, 213, 68, 7, 181, 238, 52, 154, 113, 56, 76, 86,
+          44, 245, 128, 194, 103, 14, 81, 229, 124, 189, 13, 252, 138, 98, 196,
+          218, 39, 34, 42,
```

Comment:
I feel like this will be better off going in a variable maybe  at the bottom of the script  so that the examples do not have this huge arrays. 

Chances are that people will read them from somewhere either way.